composer.json

```diff
@@ -20,11 +20,16 @@
     "require": {
         "php": "^8.0",
         "psr/http-message": "^1.0",
-        "psr/http-server-middleware": "^1.0",
         "psr/http-server-handler": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "psr/log": "^2.0",
         "sentry/sdk": "^3.2",
+        "symfony/console": "^5.4|^6.0",
         "yiisoft/di": "^1.0",
-        "symfony/console": "^5.4|^6.0"
+        "yiisoft/log": "^2.0",
+        "yiisoft/router": "^1.1",
+        "yiisoft/router-fastroute": "^1.1",
```

Comment:
Need to actualize dependencies. For example, `yiisoft/router-fastroute` seems to be unused at all. Also since tracing can be optional, maybe make some dependencies optional as well (via `suggest` section). Not sure about the latter, others' opinions are welcome.

Some useful utilities:

- https://github.com/composer-unused/composer-unused
- https://github.com/maglnet/ComposerRequireChecker (soon it will be added to all repos via actions).

Reply:
> Need to actualize dependencies. For example, `yiisoft/router-fastroute` seems to be unused at all. Also since tracing can be optional, maybe make some dependencies optional as well (via `suggest` section). Not sure about the latter, others' opinions are welcome.
i will use yiisoft/dummy-provider in dev-dependency